src/odemis/driver/hamamatsurx.py

```diff
@@ -489,6 +489,7 @@ def _getDataFromBuffer(self):
 
                 self._metadata[model.MD_ACQ_DATE] = time.time()
                 # TODO more fancy maybe? metadata[model.MD_ACQ_DATE] = time.time() - (exposure_time + readout_time)
```

Comment:
I like the more fancy case, because some part of Odemis do rely on this metadata to check whether the image is "fresh" enough. It should be relatively safe, as exposure time and readout time are never going to be huge anyway.